docs/source/install/k8s_ha.rst

```diff
@@ -0,0 +1,369 @@
+|st2| HA Cluster in Kubernetes - BETA
+=====================================
+
+This document provides an installation blueprint for a Highly Availabile StackStorm cluster
+based on `Kubernetes <https://kubernetes.io/>`__, a container orchestration platform at planet scale.
+
+The cluster deploys a minimum of 2 replicas for each component of StackStorm microservices for redundancy and reliability,
+as well as configures backends like MongoDB HA Replicaset, RabbitMQ HA and etcd cluster that st2 relies on for database,
+communication bus, and distributed coordination respectively. That raises a fleet of more than ``30`` pods total.
+
+The source code for K8s resource templates is available as a GitHub repo:
+`StackStorm/stackstorm-ha <https://github.com/StackStorm/stackstorm-ha>`_.
+
+.. warning::
+    **Beta quality!**
+    As this deployment method available in beta version, documentation and code may be substantially modified and refactored.
+
+.. contents:: Contents
+   :local:
+
+---------------------------
+
+Requirements
+------------
+* `Kubernetes <https://kubernetes.io/docs/setup/pick-right-solution/>`__ cluster
+* `Helm <https://docs.helm.sh/using_helm/#install-helm>`__, the K8s package manager and `Tiller <https://docs.helm.sh/using_helm/#initialize-helm-and-install-tiller>`_
+* Enough computing resources for production use, respecting :doc:`/install/system_requirements`
+
+Usage
+-----
+This document assumes some basic knowledge of Kubernetes and Helm.
+Please refer to `K8s <https://kubernetes.io/docs/home/>`__ and `Helm <https://docs.helm.sh/>`__
+documentation if you find any difficulties using these tools.
+
+However, here are some minimal instructions to get started.
+
+Deployment
+__________
+StackStorm HA cluster available as a Helm chart, a bundled K8s package which
```

Comment:
The StackStorm HA cluster is available as a Helm chart, a bundled K8s package which makes installing the complex StackStorm infrastructure as easy as: